data/json/furniture_and_terrain/appliances.json

```
@@ -34,6 +34,23 @@
     ],
     "damage_reduction": { "all": 32 }
   },
+  {
+    "type": "vehicle_part",
+    "id": "ap_glass_fridge",
+    "name": { "str": "glass fridge" },
+    "looks_like": "f_glass_fridge",
+    "description": "A modern refrigerator with a thick sheet of glass in the door, often specially treated to be more insulative.  Allows seeing the contents without letting out the cold air, which used to be a minor convenience, and now saves precious minutes until spoilage.",
```

```suggestion
    "description": "A modern refrigerator with a thick sheet of glass in the door, often specially treated to be more insulative.  Allows seeing the contents without letting out the cold air.",
```
I don't think we want to acknowledge in game that things instantly turn spoiled from one second to the next